README.md

```diff
@@ -96,12 +95,12 @@ let payload = {
     userAgent: "Mozilla/5.0 (iPhone; CPU iPhone OS 10_3 like Mac OS X) AppleWebKit/602.1.50 (KHTML, like Gecko) CriOS/56.0.2924.75 Mobile/14E5239e Safari/602.1"
 };
```

Comment:
can we add any info related to agreeSimilarTransaction in read me

Reply:
I've updated the readme with example and explanation.